stories/coverage.fmf

```diff
@@ -0,0 +1,88 @@
+story:
+    As a QE I want to document the test coverage so it can be easily tracked
+    which requirement is covered by which test case and vice-versa.
+
+/notes:
+    summary:
+        Story notes.
+    story:
+        As a product owner I want to add additional content to the requirements,
+        like notes or context, so extra information can be provided.
+    description:
+        There should be a parameter that has no strict requirement and
+        can be used to provide any extra information that may be necessary.
+    example:
+        - 'story:'
+        - '     As an admin I want product A doing B so I can use it for C.'
+        - 'notes:'
+        - '     The feature should help our customer to achieve D.'
+    priority: low
+
+/priority:
+    summary:
+        Story priority.
+    story:
+        As a product owner I want to prioritize the requirements
+        so the importance can be stated.
```

Comment:
```suggestion
        so that the importance can be stated.
```
I think `so that` is a bit more straightforward. General definition:

    As a <role> I can <capability>, so that <receive benefit>